File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03073/p03073_num2_parsed.cpp
#include<iostream>
#include<string>

using namespace std;

int minRepaintTiles(string s) {
    int count = 0;
    for (int i = 1; i < s.length(); i++) {
        if (s[i] == s[i - 1]) {
            count++;
        }
    }
    return count;
}

int main() {
    string S;
    cin >> S;

    int result = minRepaintTiles(S);
    cout << result << endl;

    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03073/p03073_num2_parsed.py
def min_repaint_tiles(S):
    count = 0
    for i in range(1, len(S)):
        if S[i] == S[i-1]:
            count += 1
    return count

# Read the input
S = input().strip()

# Call the function and print the result
result = min_repaint_tiles(S)
print(result)
